src/stories/informationaltooltip/InformationalTooltip.tsx

```diff
@@ -73,53 +69,83 @@ export const InformationalTooltip: FC<InformationalTooltipProps> = ({
     side = 'bottom',
     sideOffset = 6,
     align = 'start',
-    open,
-    onOpenChange,
-}) => (
-    <RadixTooltip.Provider
-        delayDuration={150}
-    >
-        <RadixTooltip.Root
-            open={open}
-            onOpenChange={onOpenChange}
+    defaultOpen = false,
+}) => {
+    const [isOpen, setOpen] = useState(defaultOpen);
+
+    /**
+     * Converts a CSS time value string to milliseconds
+     * @param timeValue - The CSS time value (e.g. '100ms', '0.5s')
+     * @returns The time value in milliseconds
+     * @throws Error if the time value format is invalid
+     */
+    const parseCSSTime = (timeValue: string): number => {
+        // Match the value and unit
+        const match = timeValue.match(/^([\d.]+)(ms|s)$/);
+
+        if (!match) {
+            throw new Error('Invalid CSS time format. Expected formats: "100ms", "0.5s"');
```

Comment:
Here we can fallback to 0 instead of throwing an error, and we could put a console.warn or something